.rubocop.yml

```diff
@@ -71,6 +71,7 @@ Rails/SkipsModelValidations:
   Exclude:
     - 'db/migrate/*.rb'
     - 'app/controllers/users_controller.rb'
+    - 'app/models/trace.rb'
```

Comment:
Why do we need to exclude this?

Reply:
Rubocop was complaining about the use of insert_all! because it's bypassing validations.